app/src/main/java/com/fs_sournary/weather/screen/main/MainViewModel.java

```diff
@@ -1,6 +1,8 @@
 package com.fs_sournary.weather.screen.main;
 
 import android.databinding.BaseObservable;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentPagerAdapter;
```

Comment:
ERROR: Unused import: android.support.v4.app.FragmentPagerAdapter. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports